cfme/configure/tasks.py

```diff
@@ -105,6 +109,167 @@ def is_analysis_finished(name, task_type='vm', clear_tasks_after_success=True):
                             clear_tasks_after_success=clear_tasks_after_success)
 
 
+def are_all_tasks_finished(name, expected_num_of_tasks, task_type='container'):
+    """ Check if all tasks states are finished - if not, reload page"""
+
+    tabs_data = {
```

Comment:
**Required**

DRY this implementation by pulling tabs_data out of these methods. With this PR the same dictionary is now copied to 3 places in this module. (one existing on ~L80, then the two you added)

Reply:
DONE!!
Pull out the tabs_data out of these methods. 
Add new TABS_DATA_PER_PROVIDER dictionary instead.
Use TABS_DATA_PER_PROVIDER on 3 methods.